packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -174,7 +174,7 @@ export function _SignInStart(): JSX.Element {
           case ERROR_CODES.SAML_USER_ATTRIBUTE_MISSING:
           case ERROR_CODES.OAUTH_EMAIL_DOMAIN_RESERVED_BY_SAML:
           case ERROR_CODES.USER_LOCKED:
-            card.setError(error.longMessage);
+            card.setError(error);
```

Comment:
@anagstef but does the code exist under the `unstableErrors`  in localizations ?

Reply:
Doesn't matter, if it does not exist, it fallbacks to the `longMessage`.